### DISCUSSION TITLE
Adding field-sizing?

### DISCUSSION THREAD

**Discussion by DavydeVries** [tailwindlabs/tailwindcss#13812] [Ideas]
Maybe it's useful to add `field-sizing`, aka:
- `field-sizing: content;`
- `field-sizing: fixed;`

But isn't supported on all browsers ([caniuse](https://caniuse.com/?search=field-sizing%3A%20content%3B)/[mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing))

This function allows the input automatically size based on the rows.

**Comment by alecfilios** [comment]
Still no news on that right?

**Reply by AMEH64** [reply]
Looks like it was added to v4.
- https://github.com/tailwindlabs/tailwindcss/pull/14469
- https://tailwindcss.com/docs/v4-beta#field-sizing-utilities

You can backfill this to v3 yourself using the plugins API.

```js
import plugin from "tailwindcss/plugin";

/** @type {import('tailwindcss').Config} */
export default {
  // ...
  plugins: [
    plugin(({ addUtilities }) => {
      addUtilities({
        ".field-sizing-content": {
          "field-sizing": "content",
        },
      });
    }),
  ],
};
```

**Comment by harellevy** [comment]
missing also field-sizing: normal
